.github/workflows/black.yml

```diff
@@ -7,4 +7,22 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: psf/black@stable
\ No newline at end of file
+      - uses: psf/black@stable
+        with:
+          options: ""
+      - name: switching from HTTPS to SSH
+        run: git remote set-url origin https://github.com/mickeyshi/altk-actions.git
```

Comment:
Can you explain this?

Reply:
Yep yep, that command just sets the origin for publishing to the repo. The HTTP-SSH comment is incongruous, though, I had taken that bit from a tutorial that was setting this up. Will make it more clear

Reply:
Right, we just don't want the target to be your repo.  It's feeling to me like we need a good way of having test/sandbox stuff separately from the main.  Will think of best way of setting this up